condition.go

```diff
@@ -117,3 +120,51 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+func buildLikeCmp(d Dialect, buf Buffer, pred string, column string, value interface{}) error {
+	if value == nil {
+		return fmt.Errorf("Column %s %s nil is Not Supported", column, pred)
```

Comment:
Please use an exported error value.

Reply:
I’m sorry, but because there was a problem with my repository management, I issued a new PR and the same content PR. And I fixed the pointed out matter.

https://github.com/gocraft/dbr/pull/103